src/v2/Apps/Auction/Components/AuctionDetails/CascadingEndTimesBanner.tsx

```diff
@@ -5,22 +5,33 @@ import { getENV } from "v2/Utils/getENV"
 
 interface CascadingEndTimesBannerProps {
   cascadingEndTimeIntervalMinutes: number
+  extendedBiddingIntervalMinutes: number | null
 }
 
 export const CascadingEndTimesBanner: React.FC<CascadingEndTimesBannerProps> = ({
   cascadingEndTimeIntervalMinutes,
+  extendedBiddingIntervalMinutes,
 }) => {
   const helpArticleLink = getENV("CASCADING_AUCTION_HELP_ARTICLE_LINK")
 
   const hasLink = !!helpArticleLink
+  function getBannerText(extendedBiddingIntervalMinutes) {
+    let bannerText
+    if (extendedBiddingIntervalMinutes) {
+      bannerText =
+        "Closing times may be extended due to last minute competitive bidding"
+    } else {
+      bannerText = `Lots close at ${cascadingEndTimeIntervalMinutes}-minute intervals`
+    }
+    return bannerText
+  }
 
   return (
     <FullBleed backgroundColor={"blue100"}>
       <AppContainer>
         <HorizontalPadding>
           <Banner dismissable pl={0} variant="brand">
-            {`Lots close at ${cascadingEndTimeIntervalMinutes}-minute intervals`}
-            .
+            {getBannerText(extendedBiddingIntervalMinutes)}.
```

Comment:
#minor rather than `{function()}`, its a bit more idiomatic IMO to render a component, so something like `<BannerText extendedBiddingIntervalMinutes={extendedBiddingIntervalMinutes} />`

Reply:
so that would essentially function as a wrapper around the cascading banner, that seems reasonable to me!